deployment/docker-compose/docker-compose.yml

```diff
@@ -71,20 +71,23 @@ services:
       /usr/local/bin/python server.py --port=5001 --models=base
     restart: always
   embeddings:
-    image: idinsight/embeddings:latest
+    image: ${EMBEDDINGS_IMAGE:-ghcr.io/huggingface/text-embeddings-inference:cpu-1.5}
     profiles:
       - embeddings
```

Comment:
update docker compose docs with the fact that this optional tag exists and how to use it, and that you need to set the env vars in the .env file and comment out the openai embeddings in the proxy config yaml